src/app/components/button/StyledButton/style.ts

```diff
@@ -0,0 +1,21 @@
+import styled from 'styled-components';
+
+interface PropsButton {
+  color?: string;
+  backgroundColor?: string;
+  borderColor?: string;
+  width?: string;
+  type?: string;
+  clickHandler(): void;
+}
+
+export const Button = styled.button`
```

Comment:
can't we use interfaces as generics, like `styled.button<PropsButton>`, like you did in `ArticleImage.ts` file?